Tests/RunCMake/CommandLine/RunCMakeTest.cmake

```diff
@@ -54,6 +54,9 @@ function(run_BuildDir)
 endfunction()
 run_BuildDir()
 
+run_cmake_command(build-multiple-targets ${CMAKE_COMMAND} -E chdir ..
+  ${CMAKE_COMMAND} --build BuildDir-build --target CustomTarget --target CustomTarget2)
+
```

Comment:
Put these two lines at the bottom of the `run_BuildDir` function so that they run in the context of that test.